R/numeric_range.R

```diff
@@ -0,0 +1,102 @@
+#' Truncate the range of numeric predictions
+#'
+#' @param x A [container()].
+#' @param upper_limit,lower_limit A numeric value, NA (for no truncation) or
+#' [hardhat::tune()].
+
+#' @export
+adjust_numeric_range <- function(x, lower_limit = -Inf, upper_limit = Inf) {
```

Comment:
The probably function accepts NA values for the limits. Function this is the same as infinite values so I might remove that "feature" from probably.